src/Executor/Executor.php

```diff
@@ -176,31 +183,23 @@ public function createExecution(
                             entrypoint: $entrypoint,
                             commands: []
                         );
-                        $response = $this->call(self::METHOD_POST, $route, $headers, $params, true, 30);
+                        $response = $this->call(self::METHOD_POST, $route, $headers, $params, true, $requestTimeout);
                         $status = $response['headers']['status-code'];
                         break;
-                    case 406:
-                        $response = $this->call(self::METHOD_POST, $route, $headers, $params, true, 30);
+                    case $status == 406:
+                        $response = $this->call(self::METHOD_POST, $route, $headers, $params, true, $requestTimeout);
                         $status = $response['headers']['status-code'];
                         break;
                     default:
                         throw new \Exception($response['body']['message'], $status);
                 }
-
-                if ($status < 400) {
-                    return $response['body'];
-                }
-                
-                if ($status != 406) {
-                    throw new \Exception($response['body']['message'], $status);
-                }
-
-                sleep(2);
+            } catch (\Exception $e) {
+                throw new \Exception($e->getMessage(), $e->getCode());
             }
-            throw new Exception($response['body']['message'], 503);
+            sleep(2);
```

Comment:
What is this timeout for?

Reply:
This timeout is used when we receive a `Runtime Not Found` error from the executor.

So we need to wait and make a couple of execution attempts to wait for the runtime to be created